seeds/breadboard/docs/examples/node-example-2.js

```diff
@@ -1,34 +0,0 @@
-/**
```

Comment:
Or that file.

Here's what I would suggest: let's change the TypeDoc out directory to point to `./docs/api`. This way, there won't be any conflicts.